.github/workflows/ci.yml

```diff
@@ -85,20 +85,17 @@ jobs:
         if: ${{ matrix.python-version == '3.11' }} 
         run: |
           python -m pip install ".[hpopt]"
-      - name: Test with pytest
-        shell: bash -l {0}
-        run: |
-          # specifically disable running on GPU, otherwise Lightning will attempt to use the Apple GPU
-          # that the GitHub actions runners still have but are not allowed to actually use
-          pytest -v tests
+      # - name: Test with pytest
+      #   shell: bash -l {0}
+      #   run: |
+      #     # specifically disable running on GPU, otherwise Lightning will attempt to use the Apple GPU
+      #     # that the GitHub actions runners still have but are not allowed to actually use
+      #     pytest -v tests
```

Comment:
Thanks for reviewing! I'm trying to debug the notebook section and I don't want to wait for 20 minute every time I test something. I will drop this if you think this PR can otherwise be merged.